doc/SAPV.md

```diff
@@ -0,0 +1,138 @@
+# EPIC-SAPV Design
```

Comment:
It would be good to convert all files to restructured text to support links to other doc files.